### ISSUE TITLE
Error message ONS

### ISSUE THREAD

**Issue by sprintv** [retrospect-addon/plugin.video.retrospect#1761]
**Describe the bug**

Retrospect really is an excellent addon. However, when I try to watch a programme from "ONS" I get the message "No episode found for the selected item". These programmes can normally be watched on their website "https://www.kijkbijons.nl/tvgemist/archief". For example "Allo Allo". Am I overlooking something or is this not working (yet)?

**Expected Behavior**
Play the programme

**Actual Behavior**
Error message "No episode found for the selected item"

**To Reproduce**
Open Retrospect
Select -> Nationaal -> Ons -> Allo Allo

**Used versions:**
- Kodi version: 20.3
- Retrospect version: 5.7.5

**Comment by basrieter** [comment]
Please add a DEBUG retrospect.log as requested in the issue template.

**Comment by sprintv** [comment]
Sorry Bas, hereby the log file. starting from line 664 I see something about "Ons".

[retrospect.log](https://github.com/retrospect-addon/plugin.video.retrospect/files/14271614/retrospect.log)

**Comment by basrieter** [comment]
It seems that no shows have videos, even on the site?

**Comment by sprintv** [comment]
The site features episodes of several series. For example, from Allo Allo
and Starsky & Hutch.

Op di 13 feb. 2024 21:57 schreef Bas Rieter ***@***.***>:

**Comment by basrieter** [comment]
Does it? Not for me: https://www.kijkbijons.nl/-allo-allo/programmas/detail?id=oR0npoPl6ptwaIdBXSYR6A==

![image](https://github.com/retrospect-addon/plugin.video.retrospect/assets/1310443/707cc9df-dd9f-4428-ad7b-7b4698fe62ec)

**Comment by sprintv** [comment]
Now I see. The links in Retrospect refer to "https://www.kijkbijons.nl/programmas." That's live TV. The episodes you can watch are in "https://www.kijkbijons.nl/tvgemist/archief." If you enter allo allo there you will get 2 episodes. Other series often have more episodes.

**Comment by basrieter** [comment]
Please try https://github.com/retrospect-addon/plugin.video.retrospect/archive/refs/heads/bug-1761.zip and see if it fixes things.

**Comment by appendixie** [comment]
Ok Bas bedankt , ik kan gelukkig weer terugkijken bij de NPO.  (1761.zip)

**Comment by basrieter** [comment]
En werkt ONS ook weer naar behoren?

**Comment by sprintv** [comment]
Geweldig Bas, ONS werkt nu prima. Leuk om die oude series weer eens te bekijken. Bedankt :-)

**Comment by sprintv** [comment]
Na de update naar versie 5.7.6 werkt ONS niet meer. Ik ben voor nu weer teruggegaan naar de 1761.zip. Ik neem aan dat de fix voor ONS in een latere update geintegreerd wordt?

**Comment by basrieter** [comment]
> Na de update naar versie 5.7.6 werkt ONS niet meer. Ik ben voor nu weer teruggegaan naar de 1761.zip. Ik neem aan dat de fix voor ONS in een latere update geintegreerd wordt?

Klopt, in de zip zitten ook de fixes van 5.7.6 verwerkt. Ik zal dit weekend zorgen dat ONS ook weer in de officiele release zitd.
